fdbclient/ServerKnobs.cpp

```diff
@@ -900,18 +900,18 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init ( CLUSTER_RECOVERY_EVENT_NAME_PREFIX,              "Master" );
 
 	// Encryption
-	init( ENABLE_ENCRYPTION,                                   false ); if ( randomize && BUGGIFY ) { ENABLE_ENCRYPTION = deterministicRandom()->coinflip(); }
+	init( ENABLE_ENCRYPTION,                                   false ); if ( randomize && BUGGIFY ) ENABLE_ENCRYPTION = true;
```

Comment:
a nit: may be done for unit-testing?

Reply:
good catch. will revert.

Reply:
sorry, actually doesn't change the behavior where we enable encryption only for part of the test runs. The `BUGGIFY` macro has built-in `deterministicRandom()` call.